app/assess/data.py

```diff
@@ -9,10 +9,8 @@
 from app.assess.models.application import Application
 from app.assess.models.fund import Fund
 from app.assess.models.round import Round
-from app.config import APPLICATION_STORE_API_HOST
-from app.config import FLASK_ROOT
-from app.config import FUND_STORE_API_HOST
-from app.config import ROUND_STORE_API_HOST
+from config import Config
+from flask import current_app
 
 
 # Fund Store Endpoints
```

Comment:
These fund store endpoints should perhaps be in DefaultConfig? (Ready to move into utils DefaultConfig)?